aospy/test/test_roundtrip.py

```diff
@@ -199,52 +220,104 @@ def test_vert_int_pressure(self):
         pass
 
 
+@unittest.skipIf(not model_files_exist(idealized_moist_rad), skip_message)
+class TestIdealizedMoistRad(TestAM2):
+    def setUp(self):
+        self.two_d_test_params = {'proj': aospy_test,
+                                  'model': idealized_moist_rad,
+                                  'run': test_idealized_moist_rad,
+                                  'var': t_surf,
+                                  'date_range': ('0003-01-01', '0006-12-31'),
+                                  'intvl_in': 'monthly',
+                                  'dtype_in_time': 'ts',
+                                  'dtype_out_vert': False,
+                                  'level': False}
+        self.three_d_test_params = {'proj': aospy_test,
+                                    'model': idealized_moist_rad,
+                                    'run': test_idealized_moist_rad,
+                                    'var': temp,
+                                    'date_range': ('0003-01-01',
+                                                   '0006-12-31'),
+                                    'intvl_in': 'monthly',
+                                    'dtype_in_time': 'ts',
+                                    'level': False}
+        self.sub_daily_test_params = {'proj': aospy_test,
+                                      'model': idealized_moist_rad,
+                                      'run': test_idealized_moist_rad,
+                                      'var': temp,
+                                      'date_range': ('0003-01-01',
+                                                     '0006-12-31'),
+                                      'intvl_in': '3-hourly',
+                                      'dtype_in_time': 'inst',
+                                      'level': False}
+
+    @unittest.skip('not tested in idealized moist rad model')
```

Comment:
Agreed.  In my latest commit, 6af1551, I redefine ```TestIdealizedMoistRad``` as extending ```TestIdealizedMoist``` rather than ```TestAM2```.  This reduces the need for re-defining the skipped methods for the second idealized model.  This gets us some of the way there within the scope of this PR.